src/Diagnostics.RuntimeHost/Services/StorageService/StorageService.cs

```diff
@@ -278,7 +278,8 @@ public async Task<int> ListBlobsInContainer()
                 options.MaximumExecutionTime = TimeSpan.FromSeconds(60);
                 OperationContext oc = new OperationContext();
                 oc.ClientRequestID = clientRequestId;
-                var blobsResult = await cloudBlobClient.ListBlobsSegmentedAsync(null, true, BlobListingDetails.All, 1000, null, options, oc);
```

Comment:
Was this not listing blobs before?

Reply:
It was. I refactored the `clobBlobClient` in my previous PR and forgot to add the statement to get container reference so it broke.